bats/tests/preferences/verify-paths.bats

```diff
@@ -0,0 +1,114 @@
+# Test case 30
+
+setup() {
+    load '../helpers/load'
+    if is_windows; then
+        skip "test not applicable on Windows"
+    fi
+    # Ensure subshells don't inherit a path that includes ~/.rd/bin
+    PATH_WITHOUT_RD=$(echo "$PATH" | tr ':' '\n' | grep -v /.rd/bin | tr '\n' ':')
```

Comment:
Because then I can't run `rdctl` without specifying an explicit path

Reply:
Really? In BATS `rdctl` is a function that will call the actual `rdctl` binary with the full path (and potentially with a `.exe` extension on Windows[^1]): https://github.com/rancher-sandbox/rancher-desktop/blob/d383b111255a62e28119185837c3f1b974f9a3a5/bats/tests/helpers/commands.bash#L72-L78

The same is true for the other bundled utilities; they all should have wrapper functions so we always call the bundled version via the full path.

So this should always work. What problem did you run into?

[^1]: We always use the Windows version of `rdctl` even in WSL because the Linux version doesn't work properly.

Reply:
Yes, you're correct. No need for that auxiliary `PATH_WITHOUT_RD` variable.